src/main/java/org/jabref/cli/JabRefCLI.java

```diff
@@ -266,7 +255,7 @@ public void printUsage() {
         String footer = '\n' + importFormatsList + outFormatsList + "\nPlease report issues at https://github.com/JabRef/jabref/issues.";
 
         HelpFormatter formatter = new HelpFormatter();
-        formatter.printHelp("jabref [OPTIONS] [BIBTEX_FILE]\n\nOptions:", header, getOptions(), footer, true);
+        formatter.printHelp(100, "jabref [OPTIONS] [BIBTEX_FILE]\n\nOptions:", header, getOptions(), footer, true);
```

Comment:
Where does the 100 come from and what does it mean?
Try to extract it to some constant, avoid magic numbers:
https://stackoverflow.com/a/47902

Reply:
Yes, so true. That 100 is the number of characters per line.
It is solved now.